tools/python/boutiques/searcher.py

```diff
@@ -44,18 +43,21 @@ def __init__(self, query, verbose=False, sandbox=False, max_results=None,
 
     def search(self):
         results = self.zenodo_search()
+        num_results = len(results.json()["hits"]["hits"])
+        total_results = results.json()["hits"]["total"]
         print_info("Showing %d of %d results."
-                   % (len(results.json()["hits"]["hits"]),
-                      results.json()["hits"]["total"]))
+                   % (num_results if num_results < self.max_results
+                      else self.max_results, total_results))
         if self.verbose:
             return self.create_results_list_verbose(results.json())
         return self.create_results_list(results.json())
 
     def zenodo_search(self):
+        # Get all results
         r = requests.get(self.zenodo_endpoint + '/api/records/?q=%s&'
                          'keywords=boutiques&keywords=schema&'
                          'keywords=version&file_type=json&type=software'
-                         '&page=1&size=%s' % (self.query, self.max_results))
+                         '&page=1&size=%s' % (self.query, 9999))
```

Comment:
Would it make sense to define a variable for 9999 in the unlikely event that this value is updated in the future?